staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logs
+
+import (
+	"fmt"
+	"github.com/go-logr/logr"
+	"github.com/spf13/pflag"
+	"k8s.io/klog/v2"
+)
+
+const logLoggingFormat = "logging-format"
+
+var supportLogFormat = []string{
+	"text",
+	"json",
+}
+
+// Options has klog format parameters
+type Options struct {
+	Logr      logr.Logger
+	logFormat string
+}
+
+// NewOptions return new klog options
+func NewOptions() *Options {
+	return &Options{}
```

Comment:
If `text` format is the default let's have `NewOptions` set `logFormat` to `text`